Setup.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 pushd "$(dirname "$0")"
 
-PINNED_CORE_SDK_VERSION=$(cat ./SpatialGDK/Extras/core-sdk.version)
+PINNED_CORE_SDK_VERSION=$(head -n 1 ./SpatialGDK/Extras/core-sdk.version)
```

Comment:
Need to update `Setup.bat` too, and everything is harder in batch :(

Reply:
batch is already taking the first line only :)

Reply:
that's a surprise, but a welcome one